services/pub/pub-downloads.service.js

```diff
@@ -0,0 +1,54 @@
+import Joi from 'joi'
+import { BaseJsonService, pathParams } from '../index.js'
+import { renderDownloadsBadge } from '../downloads.js'
+import { nonNegativeInteger } from '../validators.js'
+import { baseDescription } from './pub-common.js'
+
+const description = `${baseDescription}
+  <p>This badge shows a measure of how many developers have downloaded a package monthly.</p>`
+
+const schema = Joi.object({
+  downloadCount30Days: nonNegativeInteger,
+}).required()
+
+export default class PubDownloads extends BaseJsonService {
+  static category = 'downloads'
+
+  static route = { base: 'pub/dm', pattern: ':packageName' }
+
+  static openApi = {
+    '/pub/dm/{packageName}': {
+      get: {
+        summary: 'Pub Monthly Downloads',
+        description,
+        parameters: pathParams({
+          name: 'packageName',
+          example: 'analysis_options',
+        }),
+      },
+    },
+  }
+
+  static defaultBadgeData = { label: 'downloads' }
+
+  static render({ downloadCount30Days }) {
+    return renderDownloadsBadge({
+      downloads: downloadCount30Days,
+      interval: 'month',
+      colorOverride: 'blue',
```

Comment:
WHat's the rationale for needing a color override?

Reply:
`Pub` focuses on `Dart/Flutter`, with blue as a key element of its brand identity.

Reply:
I'd suggest not using a `colorOverride` here. The vast majority of our downloads badges don't, the few exceptions that do I suspect are due to historical reasons. Let's align for consistency with other Shields.io badges. The only compelling reason to me would have been "there's some sort of dedicated badge on the Pub website that shows the download number in blue"; in that case we might have wanted to align for consistency with that. :)

Reply:
Reverted